packages/teams-js/src/public/videoEffects.ts

```diff
@@ -17,9 +17,9 @@ export namespace videoEffects {
     : new VideoPerformanceMonitor(sendMessageToParent);
 
   /** Notify video frame processed function type */
-  type notifyVideoFrameProcessedFunctionType = () => void;
+  export type notifyVideoFrameProcessedFunctionType = () => void;
```

Comment:
Just curious what value are we adding to the documentation by add this type? Also, similar question to previous exported types which are used only in their capabilities files.

Reply:
I'm not quite sure if you are specifically asking about why there is even a type defined for `() => void` or why there's value in exporting it and documenting it? If the former, I unfortunately don't know since I didn't write this code. I could opine about whether I think there's value in defining a custom time for such a simple callback type but that's outside the scope of this PR.  😉

The value that is being added is two-fold, one having to do with "correctness" and one having to do with documentation:
1. If a particular function is exported, then it seems to reason that all the types upon which it depends (and which developers are expected to use) should also be exported.
2. From a documentation point of view, if those dependent types are not exported, then they don't show up in the documentation. For example, here is a before and after for `fullScreenChangeFunctionType`:

**Before**
![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/36546967/b6772d16-bd19-4938-82db-1e9359c250be)

Note that the `fullScreenChangeFunctionType` type listed is not clickable and that there is no documentation that describes it.

**After**
![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/36546967/893a50e2-1957-45b9-8b11-007185295cad)

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/36546967/45d12344-2509-4bad-a813-ef27d96f2011)


`fullScreenChangeFunctionType` is now clickable and has a doc section of its own to describe it

Does that answer your question?